src/features/Main/Sidebar/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Sidebar as SidebarBase, MenuSection, MenuItem } from 'react-paragon-topaz';
+
+import { updateActiveTab } from 'features/Main/data/slice';
+
+const menuItems = [
+  {
+    link: 'dashboard',
+    label: 'Home',
+    icon: <i className="fa-regular fa-house" />,
+  },
+  {
+    link: 'students',
+    label: 'Students',
+    icon: <i className="fa-solid fa-person-chalkboard" />,
```

Comment:
This is not the correct icon

Reply:
That's I what I said in the PR description:

_Note
Some icons does not match with the [figma](https://www.figma.com/design/pRMTr6L9S9oNFV3qZgyHcn/CertPrep-Manager-(Dashboard)---11.02.2023?node-id=1947-62893&t=Hp0MTo2nYQCpzvZH-0) this is because this are premium icons so I decided to choose the free tier._

As walkaround I was thinking to donwload directly from Figma and import it here as svg, what do u think bout it? @AuraAlba

Reply:
But we already include the fontawesome css and that let us to use that version, I tested local:
![Screenshot from 2024-07-11 11-39-03](https://github.com/Pearson-Advance/frontend-app-instructors-portal/assets/36944773/877e4570-5481-445c-9600-0378794e1a22)

students: className="fa-solid fa-screen-users" 
my profile: className="fa-regular fa-user-group"